### DISCUSSION TITLE
How do I use Quarto with Julia 1.11?

### DISCUSSION THREAD

**Discussion by neelsmith** [quarto-dev/quarto-cli#11126] [Q&A] [answered]
### Description

Julia 1.11 breaks the API for `Pkg. is_manifest_current()`. I thought that was resolved in this issue https://github.com/quarto-dev/quarto-cli/issues/10225 but I'm encountering exactly that problem using Julia 1.11 with quarto 1.5.57 on Mac OS 14.6.1.

?

**Comment by mcanouil** [comment]
It's solved but in 1.6 pre-release.

**Comment by cderv** [comment, marked as answer]
There is still an issue with Julia 1.11 and Quarto that is being handled. You can follow 
- https://github.com/quarto-dev/quarto-cli/issues/11013
